src/twitter.ts

```diff
@@ -169,6 +169,9 @@ export const fetchTweets = async (redis: RedisClient, username: string): Promise
             .flatMap((instruction: any) => instruction.entries)
             .map((entry: any) => entry?.content?.itemContent?.tweet_results?.result)
             .filter((tweet: any) => tweet)
+            //only accept tweets whose screen_name actually matches :username
+            //  it may be a bug or it may be advertising, but incorrect tweets have been showing up on this API endpoint
```

Comment:
```suggestion
            /**
             * Only accept Tweets whose screen_name matches username. It may be
             * a bug or it may be advertising, but incorrect Tweets have been
             * showing up on this API endpoint.
             */
```